lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -5337,14 +5337,36 @@ def model_remove_prm_hvac(model)
     end
 
     # Air loops
-    model.getAirLoopHVACs.each(&:remove)
+    model.getAirLoopHVACs.each do |air_loop|
+      # Don't remove airloops representing non-mechanically cooled systems
+      if !air_loop.additionalProperties.hasFeature('non_mechanically_cooled')
+        air_loop.remove
+      else
+        # Remove heating coil on
+        air_loop.supplyComponents.each do |supply_comp|
+          # Remove standalone heating coils
+          if supply_comp.iddObjectType.valueName.to_s.include?('OS_Coil_Heating')
+            supply_comp.remove
+          # Remove heating coils wrapped in a unitary system
+          elsif supply_comp.iddObjectType.valueName.to_s.include?('OS_AirLoopHVAC_UnitarySystem')
+            unitary_system = supply_comp.to_AirLoopHVACUnitarySystem.get
+            htg_coil = unitary_system.heatingCoil
+            if htg_coil.is_initialized
+              htg_coil = htg_coil.get
+              unitary_system.resetCoolingCoil
+              htg_coil.remove
+            end
+          end
+        end
+      end
+    end
```

Comment:
Keep air loops including non-mechanically cooled (NMC) systems in the proposed model but remove heating coils. Heating coils are added through unit heaters later on when creating the baseline system (NMC heated-only system).